patches/tModLoader/Terraria/Recipe.cs.patch

```diff
@@ -49,6 +49,7 @@
 +	internal bool needWater;
 -	public bool needLava;
 +	internal bool needLava;
++	internal bool needShimmer;
```

Comment:
There is no need to make any changes to `Recipe.cs` here. The `needX` family of conditions exist solely to reduce the number of vanilla patches, and will always be `false` after vanilla recipes have loaded (they're replaced with `Condition` entries).

Please revert all changes to this file.

Reply:
I just undid the `recipe.cs` changes. This should be okay now